src/sgame/sg_missile.cpp

```diff
@@ -40,6 +40,10 @@ static Cvar::Cvar<int> g_abuild_blobFireExtinguishRange(
 		"g_abuild_blobFireExtinguishRange",
 		"range around which a granger spit extinguish fires on the ground",
 		Cvar::NONE, 64);
+static Cvar::Cvar<float> g_firefightReward(
+		"g_firefightReward",
+		"how much credits removing fire gives a granger",
```

Comment:
Probably you want this in evolve points rather than human credits? (You'll need to do a unit conversion because G_AddCreditToClient uses human credit units.)

Reply:
To be honest, I'm confused by the idea of having 2 representations. Imo this "feature" should be dropped, especially since behind the scene it's the same crap.
Anyway, you're right.

Reply:
No, gameplay wise it's good because it feels like aliens are different. Bringing back… capitalistic considerations would be break immersion on alien side.

Reply:
not really. They're the same, it's only representation which changes. There's still that dumb conversion done in various places for basically nothing, but don't care, it's not the topic here.